### PR TITLE
Made a pip installable module

### PR DESCRIPTION
**Pull request by linwoodc3**
Thanks for the opportunity.  All you'd need to do is [follow instructions from this section of documentation](https://packaging.python.org/tutorials/distributing-packages/#packaging-your-project)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,8 +25,8 @@
     version='0.1.0',
 
     description='Simple Python module for downloading fundamental financial data from financials.morningstar.com.',
-    long_description='Good Morning is intended to be used as an extension to QSToolKit (QSTK) library. By using QSTK you can easily download historical stock market data from Yahoo Finance. You can also download fundamental financial data from Compustat. However, most individuals and institutions do not have access to Compustat. Good Morning attempts to mitigate this limitation by providing a very simple Python interface for downloading fundamental financial data from financials.morningstar.com.',
-
+    long_description=long_description,
```
Comment (petercerno) [inline setup.py]: Is this change inteded?

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (petercerno) [inline .idea/vcs.xml]: Please, remove this file.

Comment (linwoodc3) [inline .idea/vcs.xml]: No it's not; my mistake.  It's an artifact of using Pycharm.  Will remove.

File: setup.py (hunk 1 above)
Comment (linwoodc3) [inline setup.py]: No it's not.  That is a waaaay old commit.  looking at the date, this is around the time I was learning Python.  The setup.py file I pushed is totally different so I'm not understanding why this is here.  Weird.  Will try to figure out what's going on.  Sorry about that.

**Comment by linwoodc3** [comment]
I got rid of the `.idea` file and also tried to get rid of the commit changing the description.  

This is the setup.py file I'm supposed to push: https://github.com/linwoodc3/good-morning/blob/master/setup.py 

Not understanding why the old one i mde when I was first learning python is there.  A little confused on this one.

**Comment by linwoodc3** [comment]
Commit a3a87c fixes the setup.py issue.  Commit 6141b77 fixes the .idea file.

**Comment by linwoodc3** [comment]
Accepting closes #4

**Pull request merged**